### ISSUE TITLE
PDF-hul: Wrong value in trailer /Size throws an unexpected exception

### ISSUE THREAD

**Issue by BertrandCaron** [openpreserve/jhove#976]
Version used: 1.32.0

If the value of the attribute /Size in the trailer is lower than the real number of objects in the PDF, JHOVE returns an unexpected exception (error PDF-hul-157).

Shouldn't JHOVE be checking that the number in the trailer /Size attribute is the same as the number of objects referenced in the xref table? And return a specific error?

Note that other tools are more explicit than JHOVE; e.g., qpdf --check returns the following warning:

`WARNING: my.pdf: reported number of objects (XX) is not one plus the highest object number`

**Comment by carlwilson** [comment]
Hi @BertrandCaron, I'm looking at this issue currently, do you happen to have an example file for this issue? Thanks.

**Comment by BertrandCaron** [comment]
Of course! Can I send it to you in a private way? It's a copyrighted-protected one, so I can't share it openly.

**Comment by BertrandCaron** [comment]
Hi @carlwilson ! Don't you need an example any more? I can send it to you (my email address is my first name dot my last name at bnf dot fr, if you want to send me yours).
